WordPress/src/main/res/values/strings.xml

```diff
@@ -1607,14 +1605,14 @@
     <string name="comments_on_my_site">Comments on my site</string>
     <string name="likes_on_my_comments">Likes on my comments</string>
     <string name="likes_on_my_posts">Likes on my posts</string>
-    <string name="site_follows">Site follows</string>
+    <string name="site_subscribers">Site subscribers</string>
```

Comment:
"Site subscriptions" would fit better here. That's how iOS did it.
<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/2471769/823dbac4-4ffd-4f96-9342-b9ccf9a42e00" width=300>

Reply:
Thanks! You're right, that does sound better 👍